### ISSUE TITLE
Add elytra pathfinding based off of pre-existing Forge Mod

### ISSUE THREAD

**Issue by bad-lang** [PrismarineJS/mineflayer#1994]
## Is your feature request related to a problem? Please describe.
I would like to be able to use elytras when pathfinding. I find it frustrating to see that this framework is capable of 

## Describe the solution you'd like
I would like the bot to use rockets and elytras to speed up travel.

## Describe alternatives you've considered
N/A

## Additional context
Here's the github repository I have been referencing

[Bebeli's Cookie Client](https://github.com/bebeli555/CookieClient/tree/main/src/main/java/me/bebeli555/cookieclient/mods/bots/elytrabot)

**Comment by tk746** [comment]
cookie client is java and uses baritone? which is a java mod i use hack clients and play on anarchy servers and i don't think it is possible with this framework just yet

**Comment by aesthetic0001** [comment]
They said "referencing" (I'm assuming the parts from the client that use the elytra to maybe find code that may be useful and possibly able to be implemented in js) not "using" but other than that you are correct

**Comment by u9g** [comment]
Should be implemented in prismarine-physics.
